cmd/convert.go

```diff
@@ -64,6 +70,7 @@ func newConvertCmd(out io.Writer) *cobra.Command {
 
 	flags.BoolVar(&deletev2Releases, "delete-v2-releases", false, "v2 release versions are deleted after migration. By default, the v2 release versions are retained")
 	flags.IntVar(&maxReleaseVersions, "release-versions-max", 10, "limit the maximum number of versions converted per release. Use 0 for no limit")
+	flags.BoolVarP(&ignoreAlreadyMigrated, "ignore-already-migrated", "e", false, "Ignore the already migrated releases to provide idempotency")
```

Comment:
Ignore the already migrated releases to provide idempotency --> Ignore any already migrated release versions and continue migrating